OpenStack-Rabbit-Consumer/rabbit_consumer/message_consumer.py

```diff
@@ -1,281 +1,134 @@
-            openstack_api.update_metadata(
-                fields.project_id, vm_id, {"AQ_STATUS": "FAILED"}
-            )
-            raise err
-
-
-def _aq_create_host(firstip, machinename, vm_id: str, fields: AqFields):
-    aq_api.create_host(
-        fields.hostnames[0],
-        machinename,
-        firstip,
-        fields.osname,
-        fields.osversion,
-    )  # osname needs to be valid otherwise it fails - also need to pass in sandbox
-    openstack_api.update_metadata(
-        fields.project_id, vm_id, {"AQ_MACHINENAME": machinename}
+
+def _print_debug_logging(rabbit_message: RabbitMessage) -> None:
+    vm_data = VmData.from_message(rabbit_message)
+    logger.debug(
+        "Project Name: %s (%s)", rabbit_message.project_name, vm_data.project_id
+    )
+    logger.debug(
+        "VM Name: %s (%s) ", rabbit_message.payload.vm_name, vm_data.virtual_machine_id
     )
+    logger.debug("Username: %s", rabbit_message.user_name)
+    logger.debug("Hostnames: %s", vm_data.hostnames)
 
 
-def _aq_update_machine_nic(machinename):
-    logger.debug("Updating Interfaces")
-    try:
-        aq_api.update_machine_interface(machinename, "eth0")
-    except Exception as err:
-        raise RuntimeError("Failed to set default interface") from err
-    logger.debug("Creating Host")
-
-
-def _aq_add_optional_nics(hostnames, machinename, message):
-    logger.debug("Creating Interfaces2")
-    for index, ip_addr in enumerate(message.get("payload").get("fixed_ips")):
-        if index > 0:
-            interfacename = "eth" + str(index)
-            try:
-                aq_api.add_machine_interface_address(
-                    machinename,
-                    ip_addr.get("address"),
-                    interfacename,
-                    # socket.gethostbyaddr(ip.get("address"))[0])
-                    hostnames[0],
-                )
-            except Exception as err:
-                raise RuntimeError("Failed to add machine interface address") from err
-
-
-def _aq_add_first_nic(machinename, message):
-    logger.debug("Creating Interfaces")
-    for index, ip_addr in enumerate(message.get("payload").get("fixed_ips")):
-        interfacename = "eth" + str(index)
-        try:
-            aq_api.add_machine_interface(
-                machinename,
-                ip_addr.get("vif_mac"),
-                interfacename,
-                # socket.gethostbyaddr(ip.get("address"))[0])
-            )
-        except Exception as err:
-            raise RuntimeError("Failed to add machine interface") from err
-
-
-def _aq_create_machine(hostnames, message):
-    logger.debug("Creating machine")
-    try:
-        machine_name = aq_api.create_machine(
-            message,
-            hostnames[-1],
-            ConsumerConfig().aq_prefix,
-        )
-    except Exception as err:
-        raise RuntimeError("Failed to create machine") from err
+def handle_machine_delete(rabbit_message: RabbitMessage):
```

Comment:
Could we add a description for this method and the type of variable is returned?